src/libraries/System.Private.CoreLib/src/System/Globalization/NumberFormatInfo.cs

```diff
@@ -441,12 +441,12 @@ public int CurrencyNegativePattern
             get => _currencyNegativePattern;
             set
             {
-                if (value < 0 || value > 15)
+                if (value < 0 || value > 16)
```

Comment:
>16 [](http://example.com/codeflow?start=41&length=2)

would it make more sense if you use the length of the array?
[https://github.com/dotnet/runtime/blob/35a5afb63fb08edca5d04a382d0663c2e1031e7a/src/libraries/System.Private.CoreLib/src/System/Number.Formatting.cs#L307](https://github.com/dotnet/runtime/blob/35a5afb63fb08edca5d04a382d0663c2e1031e7a/src/libraries/System.Private.CoreLib/src/System/Number.Formatting.cs#L307)

Reply:
Number.s_negCurrencyFormats array is private